lms/static/sass/features/_course-experience.scss

```diff
@@ -54,13 +55,52 @@
 
     .message-header {
       font-weight: $font-semibold;
-      margin-bottom: $baseline/4;
+      margin-bottom: $baseline/2;
+      width: calc(100% - 40px)
```

Comment:
@andy-armstrong: Reminder you wanted to discuss these calcs with @HarryRein.

Reply:
Thanks @robrap. @HarryRein my preference is to not use calc as it makes understanding the layout much more complicated. Generally these things can be solved very easily with flexbox and margins. Of course, Bootstrap would make it even easier. :-)

No need to change this instance, but just an FYI.